go-app.js

```diff
@@ -150,7 +152,7 @@ go.app = function() {
 
     self.states.add('Patient_Abroad_Entry', function(name) {
       return new ChoiceState(name, {
-        question: $("Has the patient travelled outside of the country in the past 2 weeks:"),
+        question: $("Has the patient travelled outside of the country in the past 3 weeks:"),
         choices: [
           new Choice('1', $("Yes")),
           new Choice('2', $("No")),
```

Comment:
This `go-app.js` file is compiled by grunt, a command line utility that compiles all the different JS files into 1 big one that we can then upload in the UI and have Vumi Go run. When reviewing you only need to look at the files changed under `src/*` as those are compiled into this one.